recipes-kernel/linux/linux-k510_git.bb

```diff
@@ -106,6 +106,10 @@ do_shared_workdir_prepend () {
 # CVE-2025-22041: Vulnerable code not present.
 # CVE-2025-22038: Vulnerable code not present.
 # CVE-2025-22037: Vulnerable code not present.
+# CVE-2023-3079: This is false positive, not because of kernel issue.
```

Comment:
You mean the following?

```
CVE-2023-3079: This is not a kernel issue.
```

Reply:
Yes, should I fix the comment like that ?